jobs/syslog_forwarder/templates/pre-start.erb

```diff
@@ -10,10 +10,12 @@ chmod 0755 ${LOGDIR}
 mkdir -p /var/vcap/data/syslog_forwarder/buffered
 chown -R syslog:adm /var/vcap/data/syslog_forwarder/buffered
 
+mkdir -p /etc/rsyslog.d
+cp $(dirname $0)/../config/rsyslog.conf /etc/rsyslog.d/rsyslog.conf
+
 <% if p('syslog.migration.disabled') == true %>
-exit
+  service rsyslog stop
```

Comment:
Please don't stop the `rsyslog` service - it is a core service which is intentionally enabled and running on the base stemcell. Aside from other releases possibly assuming rsyslog is running, rsyslog takes responsibility for kernel+service+user syslog messages and writing them to the appropriate /var/log files which other auditing/security releases should be able to rely on.

I'd recommend removing the `/etc/rsyslog.d/rsyslog.conf` file when disabled, if present. Optimally, rename `/etc/rsyslog.d/rsyslog.conf` here (and upstream) to use a more release-specific name (e.g. `25-syslog-forwarder.conf`) to avoid ambiguity.